src/Kernel.php

```diff
@@ -85,6 +83,10 @@ private function load(
             $this->composer->install($mount->path);
         }
 
+        if ($mount->path === $this->rootDir) {
+            $this->composer->requireAutoload();
```

Comment:
we only require autoload on root mount atm, and i think we will keep this to avoid deps conflict, however we certainly need to have a better error output when there is deps in mount not available in root, not sure on how to do that for the moment, will be done in a new PR